ParkingApi/Program.cs

```diff
@@ -8,8 +10,8 @@ public static void Main(string[] args)
             var builder = WebApplication.CreateBuilder(args);
 
             // Add services to the container.
-
             builder.Services.AddControllers();
+            builder.Services.AddScoped<IParkingLotService, ParkingLotService>();
```

Comment:
the same you can do for IBankAccountService and inject it into the constructor  of the ParkingLotService